articles/postgresql/flexible-server/concepts-elastic-clusters-limitations.yml

```diff
@@ -25,74 +25,79 @@ sections:
       - question: In what regions can I use elastic clusters?
         answer: |
           Elastic clusters are an Azure Database for PostgreSQL flexible server feature and as such are available in the [same regions](https://azure.microsoft.com/explore/global-infrastructure/products-by-region/).
-          
-      - question: Elastic clusters aren't available in the region I need, what can I do?
-        answer: |
-          We're enabling other regions shortly, if you're interested in specific regions fill out [this form](https://aka.ms/preview-pg-citus).
```

Comment:
We should not direct them to a form, there are concerns around PII... Rather open a support case with Capacity management type thing, Aka this si the "same" as Flex being available in any region no different